### PR TITLE
Con 2128 setup new repos

### PR DESCRIPTION
**Pull request by Carsten89**

### CODE REVIEW

File: frontend/Plugin.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
+import SgTrackingPlugin from '@shopgate/tracking-core/plugins/Base';
```
Comment (fkloes) [inline frontend/Plugin.js:8]: This module wasn't installed to my `node_modules` - not sure right now, if this is an issue.

File: frontend/Plugin.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+      if (specialTypes.indexOf(formattedData.type) > -1) {
+        formattedData.id = this.isMerchant ? data.page.merchantUrl : data.page.shopgateUrl;
+      }
+
+      this.appHandler.viewContent(formattedData, this.cmdParams);
+
+      return false;
+    });
+
+    this.register.setCampaignWithUrl((data, raw) => {
```
Comment (fkloes) [inline frontend/Plugin.js]: A JSDoc would be nice :-)

File: frontend/Plugin.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+     * This should not do anything except return false.
+     *
+     * The only reason for it is to give a single to tracking-plugin to
+     * build a right blacklist for viewContent events, since viewContent is
+     * actually sending pageView.
+     */
+    this.register.viewContent(() => false);
+
+    /*
+     Custom events starts here
```
Comment (fkloes) [inline frontend/Plugin.js]: Please convert to a proper block comment

**Review by fkloes — CHANGES_REQUESTED** [review]
The license block is missing in some files

File: frontend/Plugin.js (hunk 1 above)
Comment (Carsten89) [reply frontend/Plugin.js:8]: Hmmm, for me it's working fine. Maybe some github issues yesterday...let's check together

File: frontend/Plugin.js (hunk 2 above)
Comment (Carsten89) [reply frontend/Plugin.js]: done

File: frontend/Plugin.js (hunk 3 above)
Comment (Carsten89) [reply frontend/Plugin.js]: done

File: package.json [hunk 4]
```diff
@@ -20,7 +20,8 @@
   ],
   "dependencies": {
     "babel-preset-latest": "6.24.1",
-    "babel-preset-stage-0": "6.24.1"
+    "babel-preset-stage-0": "6.24.1",
+    "@shopgate/tracking-core": "git+https://github.com/shopgate/tracking-core.git"
```
Comment (fkloes) [inline package.json]: Please change the url according to Flo's proposal, if the core isn't available at the time when you read this

File: frontend/index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  getRelevantConfigs,
+// eslint-disable-next-line import/no-unresolved
+} from 'Library/helpers/tracking';
```
Comment (fkloes) [inline frontend/index.js]: We need to take care, that this is imported from one of our @shopgate modules in the future

**Review by fkloes — APPROVED** [review]

**Pull request merged**